packages/core/src/components/link/link.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core'
+
+@Component({
+  tag: 'atom-link',
+  styleUrl: 'link.scss',
+  shadow: true,
+})
+export class AtomLink {
+  @Prop() color: 'primary' | 'secondary' = 'primary'
+  @Prop() type: 'anchor' | 'button' = 'anchor'
+
+  @Event() click: EventEmitter
+
+  private handleClick = (event: MouseEvent) => {
+    event.preventDefault()
+    event.stopPropagation()
+
+    return this.click.emit(event)
+  }
+
+  render() {
+    return (
+      <Host>
+        {this.type === 'anchor' ? (
+          <span class='atom-link' color={this.color}>
```

Comment:
Not rendering anchor tag as router components (usually) do it - our Next version does not do it, but it is simple to create a wrapper on Colosseum combining Link and anchor tag ([see here](https://juntossomosmais.github.io/atomium/pr-preview/pr-502/?path=/docs/react_components-link--docs))

Reply:
maybe we could receive a prop if we can use it as a simple `a` tag?

Reply:
This is a particularity of each project, considering that the purpose of this DS is to be agnostic.

I think that adding another layer of complexity to this component isn't so necessary, because it gives the choice to the developer and which routing library is he using, leaving to this component only the styling (if it will be used as a literal link anchor) or styling + semantic (if being used as a button who seems like a link)

Reply:
Yeah, including what Jowjow said, router-link and NuxtLink already render an anchor tag. As it is common for Colosseum combine Link and anchor tag, guess it is simpler to keep atom-link takes care of styling only.